.eslintrc.json

```diff
@@ -19,7 +19,10 @@
     "compilerOptions": { "target": "ES6", "module": "commonjs" },
     "max-len": ["error", { "code": 100, "ignoreStrings": true, "ignoreTemplateLiterals": true }],
     "import/no-extraneous-dependencies": ["error", {"devDependencies": true}],
-    "react/prefer-stateless-function": 0
+    "react/prefer-stateless-function": 0,
+    "react/sort-comp": 0,
+    "react/jsx-wrap-multilines": 0,
+    "react/no-unused-prop-types": 0
```

Comment:
Why are these rules set off?

Reply:
"react/prefer-stateless-function" has been set off before now
"react/sort-comp" requires that components should be sorted and after sorting it the suggested way, it keeps requesting for it to be sorted
![image](https://user-images.githubusercontent.com/27571135/50983340-932e1e80-14ff-11e9-9516-a2475daf3b7b.png)

"react/no-unused-prop-types" was set off because eslint was flahhing a property that was in use as an unused property.
![image](https://user-images.githubusercontent.com/27571135/50983420-bce74580-14ff-11e9-9230-a1e56ac0535b.png)